jina/types/document/__init__.py

```diff
@@ -205,22 +205,23 @@ def __init__(
         self._mermaid_id = random_identity()  #: for mermaid visualize id
 
     @property
-    def length(self) -> int:
+    def siblings(self) -> int:
         """
-        The number of siblings of the :class:``Document``
+        Getter for siblings.
 
-        .. # noqa: DAR201
-        :getter: number of siblings
-        :setter: number of siblings
-        :type: int
+        :return: number of chunks generated from the parent document
         """
-        # TODO(Han): rename this to siblings as this shadows the built-in `length`
 
         return self._pb_body.length
 
-    @length.setter
-    def length(self, value: int):
-        self._pb_body.length = value
+    @siblings.setter
```

Comment:
If we aim for having this enforced to be the number of siblings. This may need to be handled at ChunkSet level, where everytime a Document is added,   we may need to make sure that the complete set updates their siblings value? I am not sure if that would come at too much cost? But if we aim for having such a behavior I would enforce it.

Otherwise I would even remove this field from our Proto

Reply:
You are right, it is never used.
I think if this attribute is meaningful to someone, they can have it within the tags of the document.

Reply:
Yeah it doesn't seem to be used anywhere. And just using tags would be sufficient, as Florian said